.pre-commit-config.yaml

```diff
@@ -43,24 +43,29 @@ repos:
     name: Run make verify-gen
     entry: make verify-gen
     language: system
-    stages: [pre-push]
+    stages: [ pre-push ]
```

Comment:
Here we're adding spaces, but below we're removing them from the same line. Shouldn't we be consistent, or is there some other significance to this?

Reply:
Sorry my bad, it should be consistent across with no space.